client/cdc_client.c

```diff
@@ -41,117 +39,249 @@ struct {
 	usb_desc_list_t str0;
 	usb_desc_list_t strman;
 	usb_desc_list_t strprod;
-} cdc_common;
+
+	usb_cdc_line_coding_t lineCoding;
+
+	void *ctxUser;
+	void (*cbEvent)(int, void *);
+} cdc_common = { 0 };
```

Comment:
If variable is located in .bss section it should be zeroed by kernel. I think that in this case it is redundant.

Reply:
Ack.

Reply:
Done.